solidity/contracts/converter/ConverterUpgrader.sol

```diff
@@ -134,9 +133,8 @@ contract ConverterUpgrader is IConverterUpgrader, ContractRegistryClient {
         // new converter - get the type from the converter itself
         if (isV28OrHigherConverter(_oldConverter))
             newType = _oldConverter.converterType();
-        // old converter - if it has 1 reserve token, the type is a liquid token, otherwise the type liquidity pool
-        else if (reserveTokenCount > 1)
-            newType = 1;
+            // old converter - if it has 1 reserve token, the type is a liquid token, otherwise the type liquidity pool

```

Comment:
This looks like a bad linter decision.
Adding curly braces will probably solve this.

Reply:
Yeah, you're right. I'm trying to fix these cases here and there but missed this one. Good catch!